spring-cloud-dataflow-rest-client/src/main/java/org/springframework/cloud/dataflow/rest/client/CounterOperations.java

```diff
@@ -28,16 +28,20 @@
 
 	/**
 	 * Retrieve information about the given named counter.
+	 * @param name the name of the counter
+	 * @return the counter
```

Comment:
Given we don't specify the `resource` in other Resource domain classes, I think this is fine.